content/overview/pricing/rates.md

```diff
@@ -78,6 +78,16 @@ Free of charge and without any paperwork, anyone with a government email address
 
 [Learn more about sandboxes.]({{< relref "overview/pricing/free-limited-sandbox.md" >}})
 
+## Quota pricing
+
+Quotas are priced at $0.0033 per MB reserved per day and assessed monthly. In practice, this works out to $99 per GB per month, based on a 30-day month:
+
+`$0.0033 * 1000 MB * 30 days = $99 per GB per month`
+
+The monthly cost will vary slightly from $92.40 per GB in a 28-day month to $102.30 per GB in a 31-day month.
```

Comment:
Do our users really care about this level of precision? I can't imagine we need to specify the cents, but additionally the difference is really only $10. That's like not going to make or break an agency's budget.